acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/signer/Signer.java

```diff
@@ -204,16 +204,29 @@ public static String signPath(final KeyType keyType) {
     return keyType == BLS ? ETH2_SIGN_ENDPOINT : ETH1_SIGN_ENDPOINT;
   }
 
-  public Set<String> getMetricsMatching(final List<String> metricsOfInterest) {
+  public Map<String, String> getMatchedMetrics(final Set<String> metricsOfInterest) {
     final Response response =
         given().baseUri(getMetricsUrl()).contentType(ContentType.JSON).when().get(METRICS_ENDPOINT);
 
-    final List<String> lines =
-        Arrays.asList(response.getBody().asString().split(String.format("%n")).clone());
+    return convertToLines(response.getBody().asString()).stream()
```

Comment:
This could be simplified to:
```response.getBody().asString().lines()```

Removing the need for convertToLines

Reply:
nice suggestion, updated.